src/randoop/operation/AbstractOperation.java

```diff
@@ -0,0 +1,59 @@
+package randoop.operation;
+
+import randoop.reflection.ReflectionPredicate;
+
+/**
+ * AbstractOperation is an abstract implementation of the Operation interface
+ * to provide default implementations of Operation predicates that are false
+ * except for a few kinds of operations.
+ * 
+ * @author bjkeller
+ *
+ */
+public abstract class AbstractOperation implements Operation {
+
+  @Override
```

Comment:
For every method, please add a Javadoc comment -- at least `@inheritDoc` so it shows up in the generated HTML API documentation.